src/__tests__/patients/GeneralInformation.test.tsx

```diff
@@ -21,95 +23,76 @@ describe('General Information', () => {
-
-    const ageInput = wrapper.findWhere((w) => w.prop('name') === 'age')
-    expect(ageInput.prop('value')).toEqual('0')
-    expect(ageInput.prop('label')).toEqual('patient.age')
-    expect(ageInput.prop('isEditable')).toBeFalsy()
-  })
-
   it('should render the date of the birth of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'dateOfBirth')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'dateOfBirth')
     expect(dateOfBirthInput.prop('value')).toEqual(new Date(patient.dateOfBirth))
     expect(dateOfBirthInput.prop('label')).toEqual('patient.dateOfBirth')
     expect(dateOfBirthInput.prop('isEditable')).toBeFalsy()
   })
 
   it('should render the occupation of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'occupation')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'occupation')
     expect(dateOfBirthInput.prop('value')).toEqual(patient.occupation)
     expect(dateOfBirthInput.prop('label')).toEqual('patient.occupation')
     expect(dateOfBirthInput.prop('isEditable')).toBeFalsy()
   })
 
   it('should render the preferred language of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'preferredLanguage')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'preferredLanguage')
     expect(dateOfBirthInput.prop('value')).toEqual(patient.preferredLanguage)
     expect(dateOfBirthInput.prop('label')).toEqual('patient.preferredLanguage')
     expect(dateOfBirthInput.prop('isEditable')).toBeFalsy()
   })
 
   it('should render the phone number of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'phoneNumber')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'phoneNumber')
     expect(dateOfBirthInput.prop('value')).toEqual(patient.phoneNumber)
     expect(dateOfBirthInput.prop('label')).toEqual('patient.phoneNumber')
     expect(dateOfBirthInput.prop('isEditable')).toBeFalsy()
   })
 
   it('should render the email of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'email')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'email')
     expect(dateOfBirthInput.prop('value')).toEqual(patient.email)
     expect(dateOfBirthInput.prop('label')).toEqual('patient.email')
     expect(dateOfBirthInput.prop('isEditable')).toBeFalsy()
   })
 
   it('should render the address of the patient', () => {
-    const wrapper = setup()
-
-    const dateOfBirthInput = wrapper.findWhere((w) => w.prop('name') === 'address')
+    const dateOfBirthInput = wrapper.findWhere((w: any) => w.prop('name') === 'address')
```

Comment:
I think this variable name should be `addressTextField`. This is probably my bad, copy paste :(

Reply:
done